docs/getting-started/webhook.rst

```diff
@@ -32,47 +32,18 @@ The ValidatingWebhookConfiguration instructs the Kubernetes apiserver to
 POST the contents of any Create or Update operations performed on cert-manager
 resource types in order to validate that they are setting valid configurations.
 
-This allows us to ensure mis-configurations are caught early on and communicated
-to you.
+This allows us to ensure mis-configurations are caught early on and
+communicated to you.
 
 In order for this to work, the webhook requires a TLS certificate that the
-apiserver is configured to trust.
+apiserver is configured to trust. This is created by the webhook itself and is
+implemented by the following two Secrets:
 
-The cert-manager deployment manifests define two Issuer resources, and two
-Certificate resources:
-
-* issuer/cert-manager-webhook-selfsign - A self signing Issuer that is used
-  to issue a self signed root CA certificate.
-* certificate/cert-manager-webhook-ca - A self-signed root CA certificate
+* secret/cert-manager-webhook-ca - A self-signed root CA certificate
   which is used to sign certificates for the webhook pod.
-* issuer/cert-manager-webhook-ca - A CA Issuer that is used to issue
-  certificates used by the webhook pod to serve with.
-* certificate/cert-manager-webhook-webhook-tls - A TLS certificate issued by the
+* secret/cert-manager-webhook-tls - A TLS certificate issued by the
   root CA above, served by the webhook.
```

Comment:
If we describe these secrets, we should also explain where they come from else users may expect they are meant to provision them manually. Mention that the 'webhookbootstrap' controller is responsible for creating and renewing these certificates briefly?

Reply:
Also include something to the effect of "where to start/what to check if the you get webhook related errors"? Some users not on GKE private clusters may see this (i.e. with certain EKS configurations, or custom EC2, or 'self-hosted')